code/game/gamemodes/game_mode.dm

```diff
@@ -230,7 +230,10 @@ var/global/list/additional_antag_types = list()
 	refresh_event_modifiers()
 
 	spawn (ROUNDSTART_LOGOUT_REPORT_TIME)
-		display_roundstart_logout_report()
+		var/logout_report = get_logout_report()
+		for(var/mob/M in mob_list)

```

Comment:
Use staff list called `staff` it contains all clients of staff, tho you still need to check for perms. Also trial mods and retired admins have both of these perms, but devs don't. Usually devs also get round start logout report

Reply:
Mob list could get up to 100-200 during high pop + carp migration + dwellers + mining drone dungeon.

Reply:
Good catch (that happens when you move oldcode around).